packages/perseus/src/interactive2/__tests__/movable.test.ts

```diff
@@ -0,0 +1,182 @@
+/* eslint-disable testing-library/prefer-user-event */
+import {fireEvent} from "@testing-library/react";
+
+import {Graphie} from "../../util/graphie";
+import Movable from "../movable";
+
+// Yay for side-effect imports!
+// eslint-disable-next-line import/no-unassigned-import
+import "../../util/interactive";
+
+describe("Movable", () => {
+    it("should be constructable", () => {
+        const movable = new Movable(
+            new Graphie(document.createElement("div")),
+            {},
+        );
+
+        expect(movable).toBeDefined();
+    });
+
+    it("should have accessor properties", () => {
+        const movable = new Movable(
+            new Graphie(document.createElement("div")),
+            {},
+        );
+
+        expect(typeof movable.isHovering).toBe("function");
+        expect(typeof movable.cursor).toBe("function");
+        expect(typeof movable.added).toBe("function");
+        expect(typeof movable.isHovering).toBe("function");
+        expect(typeof movable.isMouseOver).toBe("function");
+        expect(typeof movable.isDragging).toBe("function");
+        expect(typeof movable.mouseTarget).toBe("function");
+    });
+
+    it("cloneState()", () => {
+        const movable = new Movable(
+            new Graphie(document.createElement("div")),
+            {},
+        );
+
+        expect(movable.cloneState()).toStrictEqual(
+            expect.objectContaining({
+                id: expect.any(String),
+                add: [],
+                added: true,
+                cursor: null,
+                draw: [],
+                isDragging: false,
+                isHovering: false,
+                isMouseOver: false,
+                modify: [],
+                mouseTarget: null,
+                onClick: [],
+                onMove: [],
+                onMoveEnd: [],
+                onMoveStart: [],
+                remove: [],
+            }),
+        );
+    });
+
+    it("cloneState() should copy state", () => {
+        const movable = new Movable(
+            new Graphie(document.createElement("div")),
+            {},
+        );
+
+        expect(movable.cloneState()).not.toBe(movable.state);
+    });
+
+    it("modify() resets movable to initial state", () => {
```

Comment:
I'm not sure this test really represents desired behavior. All the calls to `modify` I could find were passing the current state, with modifications as the argument. I think it's pretty strange that `modify` can be called without arguments, and it's probably not the intended use case.

Still, I think we might want to keep this test to document the legacy behavior. Can you add a comment noting that this is a [characterization test](https://en.wikipedia.org/wiki/Characterization_test)?